src/lightning/data/streaming/data_processor.py

```diff
@@ -66,17 +68,21 @@ def _get_home_folder() -> str:
     return os.getenv("DATA_OPTIMIZER_HOME_FOLDER", os.path.expanduser("~"))
 
 
+def _get_default_cache() -> str:
+    return "/cache" if _IS_IN_STUDIO else "/tmp"
```

Comment:
Might want to consider using `tempfile.gettempdir()` for a more robust definition.
See: https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir

Reply:
done !